src/components/product.js

```diff
@@ -375,6 +375,19 @@ export default class Product extends Component {
   }
 
   /**
+   * get info about product to be sent to tracker
+   * @return {Object}
+   */
+  get trackingInfo() {
+    return {
+      id: this.id,
+      name: this.model.selectedVariant.productTitle,
```

Comment:
I think it's called `name` here
https://github.com/Shopify/widget-framework/blob/master/app/assets/javascripts/framework/lib/tracking.js.coffee#L151